### PR TITLE
[ERSSUP-61464]-[]-[APIM Fix Repository Vulnerabilities]-[DMW]

### PR DESCRIPTION
**Pull request by nhsd-david-wass**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: sandbox/Makefile [hunk 1]
```diff
@@ -5,10 +5,16 @@ list:
...
+	docker build -t e-referrals-service-patient-care-api-sandbox .
+
+run:
+	docker run -p 9000:9000 e-referrals-service-patient-care-api-sandbox:latest
 
 clean:
 	rm -rf ./node_modules
+	docker stop $$(docker ps -a | awk '$$2 ~ /e-referrals-service-patient-care-api-sandbox/ {print $$1}') || /bin/true
+	docker rm $$(docker ps -a | awk '$$2 ~ /e-referrals-service-patient-care-api-sandbox/ {print $$1}') || /bin/true
+	docker rmi e-referrals-service-patient-care-api-sandbox || /bin/true
```
Comment (nhsd-david-wass) [inline sandbox/Makefile:18]: Added loocal docker commands to run sandbox on VM

Comment (nhsd-david-wass) [inline sandbox/Makefile:18]: Added loocal docker commands to run sandbox on VM

**Review by nhsd-radoslav-radev — DISMISSED** [review]
LGTM 👍

**Review by adamoldfield-nhs — APPROVED** [review]

**Pull request merged**